### DISCUSSION TITLE
Getting SSL: WRONG_VERSION_NUMBER with outlook smtp

### DISCUSSION THREAD

**Discussion by prophesional** [berrystephenw/OctoText#133] [Q&A]
I configured the smtp server as smtp.office365.outlook.com on port 587 as seen in my settings of my email account.  Logs from test emails show:

2023-01-27 17:01:44,479 - octoprint.plugins.OctoText - ERROR - Exception while talking to your mail server [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1056)
Traceback (most recent call last):
  File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_OctoText/__init__.py", line 255, in smtp_login_server
    SMTP_server = smtplib.SMTP_SSL(name, port, timeout=5)
  File "/usr/lib/python3.7/smtplib.py", line 1031, in __init__
    source_address)
  File "/usr/lib/python3.7/smtplib.py", line 251, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/lib/python3.7/smtplib.py", line 336, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/lib/python3.7/smtplib.py", line 1039, in _get_socket
    server_hostname=self._host)
  File "/usr/lib/python3.7/ssl.py", line 412, in wrap_socket
    session=session
  File "/usr/lib/python3.7/ssl.py", line 853, in _create
    self.do_handshake()
  File "/usr/lib/python3.7/ssl.py", line 1117, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1056)

any help would be appreciated.

**Comment by berrystephenw** [comment]
Try using port 456 and see if that changes what happens. Also the smtp server name looks odd. I thought it was smtp.office365.com - no outlook. I would try one of these at a time, if it doesn't work put it back and try the other.
________________________________
From: Dante Shareiff ***@***.***>
Sent: Friday, January 27, 2023 12:14 PM
To: berrystephenw/OctoText ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [berrystephenw/OctoText] Getting SSL: WRONG_VERSION_NUMBER with outlook smtp (Discussion #133)


I configured the smtp server as smtp.office365.outlook.com on port 587 as seen in my settings of my email account. Logs from test emails show:

2023-01-27 17:01:44,479 - octoprint.plugins.OctoText - ERROR - Exception while talking to your mail server [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1056)
Traceback (most recent call last):
File "/home/pi/oprint/lib/python3.7/site-packages/octoprint_OctoText/init.py", line 255, in smtp_login_server
SMTP_server = smtplib.SMTP_SSL(name, port, timeout=5)
File "/usr/lib/python3.7/smtplib.py", line 1031, in init
source_address)
File "/usr/lib/python3.7/smtplib.py", line 251, in init
(code, msg) = self.connect(host, port)
File "/usr/lib/python3.7/smtplib.py", line 336, in connect
self.sock = self._get_socket(host, port, self.timeout)
File "/usr/lib/python3.7/smtplib.py", line 1039, in _get_socket
server_hostname=self._host)
File "/usr/lib/python3.7/ssl.py", line 412, in wrap_socket
session=session
File "/usr/lib/python3.7/ssl.py", line 853, in _create
self.do_handshake()
File "/usr/lib/python3.7/ssl.py", line 1117, in do_handshake
self._sslobj.do_handshake()
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1056)

any help would be appreciated.

**Comment by prophesional** [comment]
Sorry mistype I am using smtp.office365.com.  When replacing the text
setting with the email setting its seems to work fine just for email.  But
if I use any of the verizon relays for text notifications I get the error,
although I can email the settings just fine in a normal email with
success.

On Fri, Jan 27, 2023 at 9:25 AM Stephen Berry ***@***.***>
wrote:

> Try using port 456 and see if that changes what happens. Also the smtp
> server name looks odd. I thought it was smtp.office365.com - no outlook.
> I would try one of these at a time, if it doesn't work put it back and try
> the other.
> ________________________________
> From: Dante Shareiff ***@***.***>
> Sent: Friday, January 27, 2023 12:14 PM
> To: berrystephenw/OctoText ***@***.***>
> Cc: Subscribed ***@***.***>
> Subject: [berrystephenw/OctoText] Getting SSL: WRONG_VERSION_NUMBER with
> outlook smtp (Discussion #133)
>
>
> I configured the smtp server as smtp.office365.outlook.com on port 587 as
> seen in my settings of my email account. Logs from test emails show:
>
> 2023-01-27 17:01:44,479 - octoprint.plugins.OctoText - ERROR - Exception
> while talking to your mail server [SSL: WRONG_VERSION_NUMBER] wrong version
> number (_ssl.c:1056)
> Traceback (most recent call last):
> File
> "/home/pi/oprint/lib/python3.7/site-packages/octoprint_OctoText/init.py",
> line 255, in smtp_login_server
> SMTP_server = smtplib.SMTP_SSL(name, port, timeout=5)
> File "/usr/lib/python3.7/smtplib.py", line 1031, in init
> source_address)
> File "/usr/lib/python3.7/smtplib.py", line 251, in init
> (code, msg) = self.connect(host, port)
> File "/usr/lib/python3.7/smtplib.py", line 336, in connect
> self.sock = self._get_socket(host, port, self.timeout)
> File "/usr/lib/python3.7/smtplib.py", line 1039, in _get_socket
> server_hostname=self._host)
> File "/usr/lib/python3.7/ssl.py", line 412, in wrap_socket
> session=session
> File "/usr/lib/python3.7/ssl.py", line 853, in _create
> self.do_handshake()
> File "/usr/lib/python3.7/ssl.py", line 1117, in do_handshake
> self._sslobj.do_handshake()
> ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number
> (_ssl.c:1056)
>
> any help would be appreciated.
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/berrystephenw/OctoText/discussions/133>, or
> unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AKWZQOQ7RTHYVBAOA4NFBSLWUP67PANCNFSM6AAAAAAUI5UPKM
> >.
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/berrystephenw/OctoText/discussions/133#discussioncomment-4799297>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4GPE4PQUUX6QVOVQWMXK3WUQAH7ANCNFSM6AAAAAAUI5UPKM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Danté A. Shareiff  AKA The Prophesional

**Comment by berrystephenw** [comment]
So you are using your phone number and vtex.com and that is all you are changing? essentially once you get email working changing to a phone number and vtext.com destination should be all that you need to do...
________________________________

**Comment by MatzeMuc86** [comment]
Still not working also here - what are the correct settings for outlook / office365?
